packages/angular/projects/clr-angular/src/forms/combobox/option-items.directive.ts

```diff
@@ -72,9 +72,7 @@ export class ClrOptionItems<T> implements DoCheck, OnDestroy {
     if (this._filterField) {
       this.filteredItems = this._rawItems.filter(item => {
         const objValue = (item as any)[this._filterField];
-        return objValue !== undefined
-          ? objValue.toString().toLowerCase().indexOf(this.filter.toLowerCase().toString()) > -1
-          : false;
+        return objValue ? objValue.toString().toLowerCase().indexOf(this.filter.toLowerCase().toString()) > -1 : false;
```

Comment:
It could be simplified a bit

```ts
return  ('' + objValue).toLowerCase().includes(this.filter.toLowerCase())
```

Reply:
Thank. I guess it will work, but I want to explicitly exclude the null/undefined cases. I find it more readable and easier to maintain.